src/content/reference/app-defaulting-validation/index.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Defaulting and Validation of App CRs
+description: How defaulting and validation of app CRs is implemented by app-admission-controller
+weight: 110
+type: page
+last_review_date: 2020-12-16
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-batman
+---
+
+## Overview
+
+For Giant Swarm releases using app-operator version 3.0.0 and upwards the
+defaulting and validation logic of [App CRs]({{< relref "/content/reference/management-api/apps.application.giantswarm.io.md" >}}) is enabled. This logic is provided by
+[app-admission-controller](https://github.com/giantswarm/app-admission-controller).
+You can check if your cluster has this version of app-operator by checking the
+release details in the web interface.
+
+We have not enabled the defaulting and validation for existing App CRs to avoid
+disrupting your current usage of the App Platform.
+
+## Defaulting
+
+The defaulting logic is implemented using a [mutating admission webhook](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook). It currently
+defaults the following settings in the App CR if they are not specified.
+
+- The `app-operator.giantswarm.io/version` label that determines which instance
+of app-operator processes the App CR. The value is set based on the release
+version of the cluster. e.g. `3.0.0`
+- The cluster values configmap that contains per cluster values like the base
```

Comment:
```suggestion
- The cluster values configmap - It contains per cluster values like the base
```

Reply:
Minor suggestion to separate the thing being defaulted and what the thing does. 

It tripped me up, "Where is the verb??" then I remembered this is a continuation of the sentence, "It defaults the ff settings..."